src/CritCore/Translations/Translations.js

```diff
@@ -244,7 +244,15 @@ let translations = {
       alterEgoTooltip: 'Nombre visible / Alter ego',
       icon: 'Elija un avatar',
       submit: 'Crear',
-      selectPlaceholder: 'Elija un ícono'
+      selectPlaceholder: 'Elija un ícono',
+      errors: {
+        email: 'Email inválido',
+        userName:'UserName inválido',
+        password:'Caracteres: Min 4 | Max 30',
```

Comment:
Please set this translation as dependant of config. Check other similar translations

Reply:
Done.